cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -0,0 +1,434 @@
+  };
+
+  function _bindEvents() {
+    var inputs = _list.querySelectorAll( 'input' );
+
+    $.bind( _search, {
+      input: function() {
+        _evaluate( this.value );
+      },
+      focus: function() {
+        expand();
+      },
+      blur: function() {
+        if ( !_isBlurSkipped ) {
+          collapse();
+        }
+      },
+      mousedown: function() {
+        if ( _fieldset.getAttribute( 'visibility' ) === 'hidden' ) {
+          expand();
+        }
+      },
+      keydown: function( event ) {
+        var key = event.keyCode;
+
+        if ( _fieldset.getAttribute( 'visibility' ) === 'hidden' ) {
+          expand();
+        }
+
+        if ( key === KEY_RETURN ) {
+          event.preventDefault();
+          _highlight( DIR_NEXT );
+        } else if ( key === KEY_ESCAPE ) {
+          _resetSearch();
+          collapse();
+        } else if ( key === KEY_DOWN ) {
+          _highlight( DIR_NEXT );
+        } else if ( key === KEY_TAB
+                    && !event.shiftKey
+                    && _fieldset.getAttribute( 'visibility' ) === 'visible' ) {
+          collapse();
+        }
+      }
+    } );
+
+    $.bind( _list, {
+      mousedown: function() {
+        _isBlurSkipped = true;
+      },
+      keydown: function( event ) {
+        var key = event.keyCode;
+        var checked = event.target.checked;
+
+        if ( key === KEY_RETURN ) {
+          event.preventDefault();
+          event.target.checked = !checked;
+
+          $.fire( _queryOne( event.target ), 'change', {} );
+        } else if ( key === KEY_ESCAPE ) {
+          _search.focus();
+          collapse();
+        } else if ( key === KEY_UP ) {
+          _highlight( DIR_PREV );
+        } else if ( key === KEY_DOWN ) {
+          _highlight( DIR_NEXT );
+        }
+      }
+    } );
+
+    for ( var i = 0, len = inputs.length; i < len; i++ ) {
+      $.bind( inputs[i], {
+        change: function( event ) {
+          _updateSelections( event.target.id );
+          _resetSearch();
+        }
+      } );
+    }
+  };
+
+  function _filterContains( text, input ) {
+    return RegExp( $.regExpEscape( input.trim() ), 'i' ).test( text );
+  };
+
+  function _queryOne( expr, con ) {
+    return typeCheckers.isString( expr ) ?
+      ( con || document ).querySelector( expr ) :
+      expr || null;
+  };
+
+  $.create = function( tag, options ) {
+    var element = document.createElement( tag );
+
+    for ( var i in options ) {
+      var val = options[i];
+      var ref = _queryOne( val );
+
+      if ( i === 'inside' ) {
+        ref.appendChild( element );
+      } else if ( i === 'around' ) {
+        ref.parentNode.insertBefore( element, ref );
+        element.appendChild( ref );
+      } else if ( i in element ) {
+        element[i] = val;
+      } else {
+        element.setAttribute( i, val );
+      }
+    }
+
+    return element;
+  };
+
+  $.bind = function( element, options ) {
+    if ( element ) {
```

Comment:
Is this check needed or at the point where `bind` is called can it be ensured an element will be passed in?